compiler/literate/html.ml

```diff
@@ -75,6 +75,7 @@ let wrap_html
   Format.fprintf fmt
     "<head>\n\
      <style>\n\
+     summary { cursor: pointer; font-style: italic; }\n\
```

Comment:
Hum that styling should rather be put in the `catala-website/` folder, because by putting it here you're choosing this style for all reusers of the HTML produced. Generally CSS styles specified right in the HTML take precedence over styles specified in a separate CSS file.